### ISSUE TITLE
Update Hammer options for new additions

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#4547]
```
E           AssertionError: 
E           hammer activation-key add-host-collection
E             Removed options:
E               * host-collection-organization-id
E           
E           hammer activation-key content-override
E             Added options:
E               * content-overrides
E           
E           hammer activation-key host-collections
E             Added options:
E               * full-result
E           
E           hammer activation-key list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer activation-key remove-host-collection
E             Removed options:
E               * host-collection-organization-id
E           
E           hammer activation-key subscriptions
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer auth login
E             Added options:
E               * username
E               * password
E           
E           hammer bootdisk
E             Removed subcommands:    * subnet
E           hammer capsule create
E             Added options:
E               * download-policy
E           
E           hammer capsule update
E             Added options:
E               * download-policy
E           
E           hammer compute-resource create
E             Added options:
E               * datacenter
E               * caching-enabled
E           
E           hammer compute-resource image create
E             Added options:
E               * password
E           
E           hammer compute-resource image update
E             Added options:
E               * password
E           
E           hammer compute-resource update
E             Added options:
E               * datacenter
E               * caching-enabled
E           
E           hammer config-group (new command)
E             Added subcommands:
E               * info
E               * create
E               * list
E               * update
E               * delete
E             Added options:
E               * help
E           
E           hammer config-group create (new command)
E             Added options:
E               * puppet-classes
E               * puppet-class-ids
E               * help
E               * name
E           
E           hammer config-group delete (new command)
E             Added options:
E               * help
E               * name
E               * id
E           
E           hammer config-group info (new command)
E             Added options:
E               * help
E               * name
E               * id
E           
E           hammer config-group list (new command)
E             Added options:
E               * search
E               * order
E               * help
E               * page
E               * per-page
E           
E           hammer config-group update (new command)
E             Added options:
E               * name
E               * puppet-classes
E               * puppet-class-ids
E               * id
E               * new-name
E               * help
E           
E           hammer content-view
E             Added subcommands:
E               * component
E           
E           hammer content-view add-version
E             Added options:
E               * content-view-id
E               * content-view
E           
E           hammer content-view component (new command)
E             Added subcommands:
E               * add
E               * list
E               * update
E               * remove
E             Added options:
E               * help
E           
E           hammer content-view component add (new command)
E             Added options:
E               * component-content-view-id
E               * component-content-view
E               * help
E               * organization-label
E               * component-content-view-version
E               * component-content-view-version-id
E               * composite-content-view-id
E               * organization-id
E               * composite-content-view
E               * organization
E               * latest
E           
E           hammer content-view component list (new command)
E             Added options:
E               * help
E               * organization-label
E               * composite-content-view-id
E               * organization-id
E               * composite-content-view
E               * organization
E           
E           hammer content-view component remove (new command)
E             Added options:
E               * component-content-view-ids
E               * component-ids
E               * organization-label
E               * composite-content-view-id
E               * organization-id
E               * composite-content-view
E               * organization
E               * component-content-views
E               * help
E           
E           hammer content-view component update (new command)
E             Added options:
E               * component-content-view-id
E               * component-content-view
E               * help
E               * organization-label
E               * component-content-view-version
E               * component-content-view-version-id
E               * composite-content-view-id
E               * organization-id
E               * composite-content-view
E               * organization
E               * id
E               * latest
E           
E           hammer content-view copy
E             Added options:
E               * new-name
E           
E           hammer content-view filter list
E             Added options:
E               * full-result
E               * types
E             Removed options:
E               * full-results
E           
E           hammer content-view filter rule create
E             Added options:
E               * arch
E               * uuid
E               * names
E           
E           hammer content-view filter rule list
E             Added options:
E               * search
E               * by
E               * full-result
E               * order
E               * page
E               * per-page
E           
E           hammer content-view list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer content-view publish
E             Added options:
E               * force-yum-metadata-regeneration
E           
E           hammer content-view puppet-module list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer content-view remove-version
E             Added options:
E               * content-view-id
E               * content-view
E           
E           hammer content-view version
E             Added subcommands:
E               * republish-repositories
E           
E           hammer content-view version list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer content-view version promote
E             Added options:
E               * force-yum-metadata-regeneration
E               * description
E               * environment-ids
E           
E           hammer content-view version republish-repositories (new command)
E             Added options:
E               * help
E               * organization-label
E               * version
E               * content-view-id
E               * organization-id
E               * async
E               * organization
E               * id
E               * content-view
E           
E           hammer discovery facts
E             Added options:
E               * search
E               * per-page
E               * page
E               * order
E           
E           hammer discovery_rule create
E             Added options:
E               * organization-ids
E               * organizations
E               * location-ids
E               * locations
E           
E           hammer discovery_rule update
E             Added options:
E               * organization-ids
E               * organizations
E               * location-ids
E               * locations
E           
E           hammer docker manifest list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer docker tag list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer domain info
E             Added options:
E               * show-hidden-parameters
E           
E           hammer domain set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer environment sc-params
E             Added options:
E               * show-hidden
E           
E           hammer erratum list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer filter create
E             Added options:
E               * override
E           
E           hammer filter update
E             Added options:
E               * override
E           
E           hammer global-parameter list
E             Added options:
E               * show-hidden
E           
E           hammer global-parameter set
E             Added options:
E               * hidden-value
E           
E           hammer gpg list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer host
E             Added subcommands:
E               * rebuild-config
E           
E           hammer host create
E             Added options:
E               * pxe-loader
E               * config-groups
E               * content-source-id
E               * config-group-ids
E             Removed options:
E               * root-pass
E           
E           hammer host errata list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer host info
E             Added options:
E               * show-hidden-parameters
E           
E           hammer host interface create
E             Added options:
E               * subnet6-id
E               * ip6
E           
E           hammer host interface update
E             Added options:
E               * subnet6-id
E               * ip6
E           
E           hammer host list
E             Added options:
E               * include
E           
E           hammer host package list
E             Added options:
E               * search
E               * by
E               * full-result
E               * per-page
E               * order
E               * page
E           
E           hammer host rebuild-config (new command)
E             Added options:
E               * only
E               * help
E               * name
E               * id
E           
E           hammer host sc-params
E             Added options:
E               * show-hidden
E           
E           hammer host set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer host smart-variables
E             Added options:
E               * show-hidden
E           
E           hammer host update
E             Added options:
E               * pxe-loader
E               * config-groups
E               * content-source-id
E               * config-group-ids
E             Removed options:
E               * root-pass
E           
E           hammer host-collection hosts
E             Added options:
E               * include
E           
E           hammer host-collection list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer host-collection update
E             Removed options:
E               * new-organization-id
E           
E           hammer hostgroup create
E             Added options:
E               * pxe-loader
E               * compute-profile
E               * query-organization-id
E               * query-organization-label
E               * query-organization
E               * compute-profile-id
E               * kickstart-repository-id
E             Removed options:
E               * organization
E               * organization-id
E           
E           hammer hostgroup sc-params
E             Added options:
E               * show-hidden
E           
E           hammer hostgroup set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer hostgroup smart-variables
E             Added options:
E               * show-hidden
E           
E           hammer hostgroup update
E             Added options:
E               * pxe-loader
E               * compute-profile
E               * query-organization-id
E               * query-organization-label
E               * query-organization
E               * compute-profile-id
E               * kickstart-repository-id
E             Removed options:
E               * organization
E               * organization-id
E           
E           hammer lifecycle-environment list
E             Added options:
E               * search
E               * by
E               * full-result
E               * per-page
E               * order
E               * page
E           
E           hammer location create
E             Added options:
E               * medium-ids
E               * ignore-types
E             Removed options:
E               * media-ids
E           
E           hammer location info
E             Added options:
E               * show-hidden-parameters
E           
E           hammer location set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer location update
E             Added options:
E               * medium-ids
E               * ignore-types
E             Removed options:
E               * media-ids
E           
E           hammer organization
E             Added subcommands:
E               * add-location
E               * remove-location
E           
E           hammer organization add-location (new command)
E             Added options:
E               * location-id
E               * location
E               * help
E               * name
E               * id
E           
E           hammer organization create
E             Added options:
E               * medium-ids
E             Removed options:
E               * media-ids
E           
E           hammer organization list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer organization remove-location (new command)
E             Added options:
E               * location-id
E               * location
E               * help
E               * name
E               * id
E           
E           hammer organization set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer organization update
E             Added options:
E               * medium-ids
E               * ignore-types
E             Removed options:
E               * media-ids
E           
E           hammer os info
E             Added options:
E               * show-hidden-parameters
E           
E           hammer os list
E             Added options:
E               * os-parameters-attributes
E           
E           hammer os set-parameter
E             Added options:
E               * hidden-value
E           
E           hammer ostree-branch list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer package list
E             Added options:
E               * full-result
E               * host
E               * packages-restrict-upgradable
E               * packages-restrict-applicable
E               * host-id
E             Removed options:
E               * content-view-id
E               * full-results
E               * content-view
E           
E           hammer package-group list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer product list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer proxy create
E             Added options:
E               * download-policy
E           
E           hammer proxy update
E             Added options:
E               * download-policy
E           
E           hammer puppet-class sc-params
E             Added options:
E               * show-hidden
E           
E           hammer puppet-class smart-variables
E             Added options:
E               * show-hidden
E           
E           hammer puppet-module list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer realm (new command)
E             Added subcommands:
E               * info
E               * create
E               * list
E               * update
E               * delete
E             Added options:
E               * help
E           
E           hammer realm create (new command)
E             Added options:
E               * organizations
E               * realm-type
E               * name
E               * organization-ids
E               * location-ids
E               * locations
E               * realm-proxy-id
E               * help
E           
E           hammer realm delete (new command)
E             Added options:
E               * help
E               * name
E               * id
E           
E           hammer realm info (new command)
E             Added options:
E               * help
E               * name
E               * id
E           
E           hammer realm list (new command)
E             Added options:
E               * search
E               * help
E               * page
E               * location
E               * organization-id
E               * organization
E               * location-id
E               * order
E               * per-page
E           
E           hammer realm update (new command)
E             Added options:
E               * organizations
E               * realm-type
E               * name
E               * organization-ids
E               * location-ids
E               * locations
E               * realm-proxy-id
E               * id
E               * new-name
E               * help
E           
E           hammer repository create
E             Added options:
E               * upstream-username
E               * upstream-password
E               * verify-ssl-on-sync
E           
E           hammer repository list
E             Added options:
E               * full-result
E               * ostree-branch-id
E               * ostree-branch
E             Removed options:
E               * full-results
E           
E           hammer repository remove-content
E             Added options:
E               * sync-capsule
E               * product
E               * product-id
E               * ids
E             Removed options:
E               * content-ids
E           
E           hammer repository synchronize
E             Added options:
E               * validate-contents
E               * skip-metadata-check
E           
E           hammer repository update
E             Added options:
E               * upstream-username
E               * upstream-password
E               * verify-ssl-on-sync
E           
E           hammer repository-set list
E             Added options:
E               * search
E               * by
E               * full-result
E               * per-page
E               * order
E               * page
E           
E           hammer role
E             Added subcommands:
E               * clone
E           
E           hammer role clone (new command)
E             Added options:
E               * organizations
E               * description
E               * organization-ids
E               * location-ids
E               * locations
E               * name
E               * id
E               * new-name
E               * help
E           
E           hammer role create
E             Added options:
E               * organization-ids
E               * organizations
E               * location-ids
E               * description
E               * locations
E           
E           hammer role info
E             Added options:
E               * description
E           
E           hammer role update
E             Added options:
E               * organization-ids
E               * organizations
E               * location-ids
E               * description
E               * locations
E           
E           hammer sc-param add-override-value
E             Added options:
E               * omit
E           
E           hammer sc-param info
E             Added options:
E               * show-hidden
E           
E           hammer sc-param list
E             Added options:
E               * show-hidden
E           
E           hammer sc-param update
E             Added options:
E               * omit
E           
E           hammer smart-variable add-override-value
E             Added options:
E               * omit
E           
E           hammer smart-variable delete
E             Added options:
E               * variable
E           
E           hammer smart-variable info
E             Added options:
E               * variable
E               * show-hidden
E           
E           hammer smart-variable list
E             Added options:
E               * show-hidden
E           
E           hammer smart-variable update
E             Added options:
E               * new-variable
E             Removed options:
E               * name
E               * new-name
E           
E           hammer subnet create
E             Added options:
E               * discovery-id
E               * subnet-parameters-attributes
E               * network-type
E           
E           hammer subnet info
E             Added options:
E               * show-hidden-parameters
E           
E           hammer subnet update
E             Added options:
E               * discovery-id
E               * subnet-parameters-attributes
E               * network-type
E           
E           hammer subscription delete-manifest
E             Removed options:
E               * content-host
E               * content-host-id
E           
E           hammer subscription list
E             Added options:
E               * full-result
E             Removed options:
E               * full-results
E           
E           hammer sync-plan list
E             Added options:
E               * search
E               * by
E               * full-result
E               * per-page
E               * order
E               * page
E           
E           hammer user create
E             Added options:
E               * default-organization
E               * description
E               * default-location
E           
E           hammer user update
E             Added options:
E               * default-organization
E               * current-password
E               * description
E               * default-location
```

**Comment by lpramuk** [comment]
- [ ] --full-results renamed to --full-result

**Comment by ntkathole** [comment]
Closed via PR #5748
